### PR TITLE
Cleans up seeding script and sets up proper api route

### PR DESCRIPTION
**Pull request by kmorgan8588**

### CODE REVIEW

File: db/Schemas/Listing.js [hunk 1]
```diff
@@ -3,7 +3,7 @@ const mongoose = require('mongoose');
 
 
 const houseSchema = new mongoose.Schema({
-    Listing_id: Number,
```
Comment (pbennettwa) [inline db/Schemas/Listing.js:6]: I did the same thing.

Comment (kmorgan8588) [reply db/Schemas/Listing.js:6]: Had to change it to a string to maintain the padding 👍

File: server/index.js [hunk 2]
```diff
@@ -9,13 +9,17 @@ app.use(express.json());
```
Comment (pbennettwa) [inline server/index.js:9]: I like the way you set up your routes, I'll re-evaluate mine for complexity after this

**Review by pbennettwa — APPROVED** [review]

**Review by ghost — APPROVED** [review]
Looks bomb, copied your routes setup. Stellar.

**Pull request merged**